bitnami/phpmyadmin/Chart.yaml

```
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: phpmyadmin
-version: 6.3.3
+version: 6.3.4
```

As it is not bug fixing but a new feature, can you bump the minor version instead?
```suggestion
version: 6.4.0
```